_posts/2018-06-01.md

```diff
@@ -0,0 +1,199 @@
+- Built in LSP support (**@tjdevries**, [#6856](https://github.com/neovim/neovim/pull/6856)) (see http://langserver.org/ why that could be of interest to you)
+
+***PSA***: If you build neovim from the latest master, always check [Following HEAD](https://github.com/neovim/neovim/wiki/Following-HEAD) for any changes.
+
+
+Around Neovim
+-------------
+
+### GSOC
+
+Good news everyone! Neovim has been accepted into the [Google Summer of Code(GSOC)](https://summerofcode.withgoogle.com/). We've got two students working on interesting developments:
+
+- @UtkarshMe will be working on [extending the UI protocol and externalizing window layout and control](https://github.com/neovim/neovim/issues/8320). He's been [getting his feet wet](https://github.com/neovim/neovim/pulls?utf8=%E2%9C%93&q=author%3AUtkarshMe+) before, and will be mentored by the ever restless @bfredl. Be sure to check out [his blog](https://utkarshme.github.io/blog/google-summer-of-code-with-neovim/) for up-to-date insights!
+- @b-r-o-c-k has been [active on the windows front](https://github.com/neovim/neovim/pulls?utf8=%E2%9C%93&q=author%3Ab-r-o-c-k+) before, and is now aiming to implement a [.NET API client](https://github.com/neovim/neovim/issues/8337) for Neovim. Our beloved BPLFHLHWTU<sup>1</sup> @justinmk will be taking care of this project!
+
+<sup>1</sup> Benevolent Project Leader For However Long He Will Tolerate Us
+
+## Related projects
+
+The wiki page of [related
+projects](https://github.com/neovim/neovim/wiki/Related-projects) is an evergrowing list of projects that somehow make use of Neovim's extended capabilities. Here's a gist of additions since the last newsletter: 
+
+### Plugins
+
+The following plugins have found their way into our [wiki](https://github.com/neovim/neovim/wiki):
+
+- [neomake-multiprocess](https://github.com/tracyone/neomake-multiprocess): A vim plugin for running multiple process asynchronously base on neomake
+- [neopipe](https://github.com/coachshea/neo-pipe): send lines of text to an external command and display output in a scratch buffer
+- [nuake](https://github.com/Lenovsky/nuake): A Quake-style terminal panel for Neovim
+- [semshi](https://github.com/numirias/semshi): Semantic Highlighting for Python
+- [vim-ghost](https://github.com/raghur/vim-ghost): Neovim client for GhostText browser extension ([firefox](https://addons.mozilla.org/en-US/firefox/addon/ghosttext/) and [chrome](https://chrome.google.com/webstore/detail/ghosttext/godiecgffnchndlihlpaajjcplehddca))
+
+
+### API Clients
+
+If you thought 3 C++ clients wasn't enough, you will be happy to hear that a 4th contender has entered the arena:
+
+- [magnum-neovim-api](https://github.com/Squareys/magnum-neovim-api)
+
+If you happen to prefer a language with no API client listed, a good starting point are the [docs](https://neovim.io/doc/user/msgpack_rpc.html#RPC), and be sure to come talk about it on the [gitter channel](gitter.im/neovim).
```

Comment:
Same comment again here, add the https:// prefix, otherwise the browser thinks this is a relative link.
i.e. gitter.im/neovim -> https://gitter.im/neovim

Reply:
Thanks :)